appsmithorg/appsmith
# fix: gsheet file picker background issue fixed

## Description
This PR fixes:
- When creating google sheets datasource with specific sheets modality, the file picker is being shown on blank page, but before we see file picker, we see appsmith page for a bit and then see the blank page, this creates the illusion that file picker is being opened outside of appsmith.


https://github.com/appsmithorg/appsmith/assets/30018882/616fd98f-7193-47ce-b09a-00cf4a690ae1

- Issue fixed looks like below:


https://github.com/appsmithorg/appsmith/assets/30018882/515f8ba3-997f-4f7c-9f3b-b3c16bbff086


#### PR fixes following issue(s)
Fixes #24046 
> if no issue exists, please create an issue and ask the maintainers about this first
>
>
#### Media
> A video or a GIF is preferred. when using Loom, don’t embed because it looks like it’s a GIF. instead, just link to the video
>
>
#### Type of change
- Bug fix (non-breaking change which fixes an issue)
>
>
>
## Testing
>
#### How Has This Been Tested?
> Please describe the tests that you ran to verify your changes. Also list any relevant details for your test configuration.
> Delete anything that is not relevant
>
>
#### Test Plan
1) Check on the scope of the permission
2) Check on the file picker is appearing 
3) Check if the header of app smith is visible when the file picker is displayed to the user 

#### Issues raised during DP testing
> Link issues raised during DP testing for better visiblity and tracking (copy link from comments dropped on this PR)
>
>
>
## Checklist:
#### Dev activity


#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Test-plan-implementation#speedbreaker-features-to-consider-for-every-change) have been covered
- [ ] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans/_edit#areas-of-interest)
